front/src/common/ReleaseInformations/LicenseAttributions.tsx

```diff
@@ -2,57 +2,95 @@ import { useTranslation } from 'react-i18next';
 
 import attributionsLicenses from './json/licenses.json';
 
-type attributionsLicensesType = {
+type AttributionLicense = {
   name: string;
   version: string;
   copyright: string;
   publisher: string;
   url?: string;
-  licenses: string;
+  license: string;
 };
 
-const LicenseAttributions = () => {
-  const { t } = useTranslation('home/navbar');
-
-  // mglTraffic is not a lib from package.json, so we add it manually
-  const mglTrafficAttribution: attributionsLicensesType = {
-    licenses: 'CC-BY-NC-SA 3.0',
+// Libs which are not in the package.json, so we add it statically
+const DATA_SOURCES: AttributionLicense[] = [
+  {
+    license: 'https://geoservices.ign.fr/cgu-licences',
+    publisher: 'IGN',
+    copyright: '© IGN - 2021',
+    version: '',
+    name: 'IGN Ortho',
+    url: 'https://geoservices.ign.fr/services-web-experts-ortho',
+  },
+  {
+    license: 'https://data.sncf.com/pages/licence/#A1',
+    publisher: 'SNCF',
+    copyright: '',
+    version: '',
+    name: 'Open data SNCF',
+    url: 'https://ressources.data.sncf.com/pages/accueil/',
+  },
+  {
+    license: 'CC-BY-NC-SA 3.0',
     publisher: 'Marc Le Gad',
     copyright: '',
     version: '',
     name: 'MLG Traffic',
     url: 'http://www.mlgtraffic.net',
-  };
+  },
+  {
+    license: 'https://github.com/tilezen/joerd/blob/master/docs/attribution.md',
+    publisher: 'Mapzen',
+    copyright: '',
+    version: '',
+    name: 'Terrain Tiles',
+    url: 'https://registry.opendata.aws/terrain-tiles/',
+  },
+];
+
+type LicenseCardProps = { attribution: AttributionLicense };
+
+const LicenseCard = ({
+  attribution: { name, version, copyright, publisher, url, license },
+}: LicenseCardProps) => (
+  <a key={name} href={url} rel="noreferrer">
+    <h3 className="d-flex mr-1 mb-0">
+      {name}
+      <small className="d-flex align-items-center ml-2">
+        {`${version}`}
+        {url && <i className="ml-2 icons-external-link" />}
+      </small>
+    </h3>
+    <div className="small ml-4 mb-2">
+      {copyright.replace('(c)', '©') || publisher || `${name} authors`}
+      {license && ` / ${license}`}
+    </div>
+  </a>
+);
+
+const LicenseAttributions = () => {
+  const { t } = useTranslation('home/navbar');
 
-  const attributionsArray = [
-    ...(Object.values(attributionsLicenses) as attributionsLicensesType[]),
-    mglTrafficAttribution,
-  ];
+  const attributionsArray = [...(Object.values(attributionsLicenses) as AttributionLicense[])];
 
   const attributions = attributionsArray
```

Comment:
Should this be in a `useMemo` ?

Reply:
it's generated at build, don't think so

Reply:
I agree with Yoh, a useMemo is not necessary here I think, as the component will not re-render often